src/tool_use/scripts/activity_tracker.py

```diff
@@ -0,0 +1,609 @@
+from pathlib import Path
+import sqlite3
+import time
+import datetime
+from typing import Optional, Tuple, List
+from rich.console import Console
+from rich.table import Table
+from ..utils.ai_service import AIService
+
+HELP_TEXT = """Usage: ai log [command] [args]
```

Comment:
Help text might not belong here, but I thought keeping it close to the code it explains keeps is easy to update for now